comms/Attribute.cpp

```diff
@@ -487,9 +487,9 @@ class CAttribute : public CAttributeBase, public IAttribute, public IAttributeHi
     }
 #endif
 
-    int PreProcess(PREPROCESS_TYPE action, const TCHAR * overrideEcl, IAttributeVector & attrs, Dali::CEclExceptionVector & errs) const
+    int PreProcess(PREPROCESS_TYPE action, const TCHAR * overrideEcl, IAttributeVector & attrs, std::map<std::pair<PREPROCESS_TYPE, std::_tstring>, bool> & attrProcessed, Dali::CEclExceptionVector & errs) const
```

Comment:
Minor:  Add a typedef for this declaration (similar to wht we do for IAttributeVector)